specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-10-15-preview/EventGrid.json

```diff
@@ -6495,6 +6670,90 @@
         }
       }
     },
+    "StaticDeliveryAttributeMappingProperties": {
+      "description": "Properties of static delivery attribute mapping.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Value of the delivery attribute.",
+          "type": "string"
+        },
+        "isSecret": {
+          "description": "Boolean flag to tell if the attribute contains sensitive information .",
+          "type": "boolean"
+        }
+      }
+    },
+    "StaticDeliveryAttributeMapping": {
```

Comment:
Where is this referenced?

Reply:
@j5lim This is a derived class of "DeliveryAttributeMapping" which is being referenced in "EventHubEventSubscriptionDestinationProperties" (line 6824)